docs/extend/extend-cli/cli-setting-up.md

```diff
@@ -5,43 +5,43 @@ Before you follow the development tutorials for creating a Zowe&trade; CLI plug-
 [Install Zowe CLI](../../user-guide/cli-installcli#methods-to-install-zowe-cli).
 
 ## Initial setup
-To create your development space, clone and build [zowe-cli-sample-plugin](https://github.com/zowe/zowe-cli-sample-plugin/#zowe-cli-sample-plug-in) from source.
 
-Before you clone the repository, create a local development folder named `zowe-tutorial`. You will clone and build all projects in this folder.
+### Clone and build your project
 
-## Branches
+1. Create a local development folder named `zowe-tutorial` to clone and build all projects in this folder.
 
-There are two branches in the repository that correspond to different Zowe CLI versions. You can develop two branches of your plug-in so that users can install your plug-in into `@latest` or `@zowe-v2-lts` CLI. Developing for both versions will let you take advantage of new core features quickly and expose your plug-in to a wider range of users.
+2. Clone [`zowe-cli-sample-plugin`](https://github.com/zowe/zowe-cli-sample-plugin/#zowe-cli-sample-plug-in) and build from source.
 
-The `master` branch of Sample Plug-in is compatible with the `@zowe-v2-lts` version of core CLI (Zowe LTS release).
+    Clone the repository into your development folder to match the following structure:
+    ```
+    zowe-tutorial
+    └── zowe-cli-sample-plugin
+    ```
 
-The `master` branch of Sample Plug-in is also compatible with the `@latest` version of core CLI (Zowe Active Development release) at this time.
+    1. Open a terminal and enter `cd zowe-tutorial` to change directory into your `zowe-tutorial` folder.
+    2. Enter `git clone https://github.com/zowe/zowe-cli-sample-plugin` to clone the `zowe-cli-sample-plugin` repository.
+    3. Enter `cd zowe-cli-sample-plugin` to change directory into your `zowe-cli-sample-plugin` folder.
+    4. Enter `git checkout master` navigate to the `master` branch.
+    5. Enter `npm install` to install all dependencies and modules for the project.
+    6. Enter `npm run build` to create a production build.
+    
+    **[are steps 1-6 correct?]**
 
-For more information about the versioning scheme, see [Maintainer Versioning](https://github.com/zowe/zowe-cli/blob/master/docs/MaintainerVersioning.md) in the Zowe CLI repository.
+### Optional step: Run the automated tests
+**[is "the" really needed in Line 28?]**
```

Comment:
@adam-wolfe please respond

Reply:
@adam-wolfe - I didn't word my question well. I guess I was looking to see if the tests were specific tests (in which case the "the" would've made sense to keep). I removed the "the."